src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -601,6 +601,18 @@ public override Task InvokeConnectionsAsync(IReadOnlyList<string> connectionIds,
             return Task.WhenAll(publishTasks);
         }
 
+        public override Task InvokeUsersAsync(IReadOnlyList<string> userIds, string methodName, object[] args)
+        {
+            var message = new RedisInvocationMessage(methodName, args);
+            var publishTasks = new List<Task>();
```

Comment:
I'm guessing there is no length on `IReadOnlyList`?

Reply:
There should be. So yeah, you could initialize the List with the Length of `userIds` to save on having to grow the list as items are added.